include/bluetooth/scan.rst

```diff
@@ -3,45 +3,54 @@
 Scanning module
 ###############
 
-The Scanning Module handles the BLE scanning for your application. You can use it to find an advertising device and establish a connection with it. The scan can be narrowed down to the device of a specific type by using filters.
+The scanning module handles the Bluetooth Low Energy scanning for your application.
+You can use it to find an advertising device and establish a connection with it.
```

Comment:
Can find many devices, change "an advertising device" to "advertising devices"?

Reply:
fixed